### PR TITLE
Metawrap prerequisites

### PR DESCRIPTION
**Pull request by ursky**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/metabat2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% set name = "metabat2" %}
+{% set version = "2.12.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://bitbucket.org/berkeleylab/metabat/downloads/metabat-static-binary-linux-x64_v2.12.1.tar.gz
```
Comment (rvalieris) [inline recipes/metabat2/meta.yaml]: use the `{{ version }}` variable here instead of a fixed version.

File: recipes/metabat2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  md5: 4c8345ba5b3023b7b4cd16b38bd70c64
+
+build:
+  number: 1
```
Comment (rvalieris) [inline recipes/metabat2/meta.yaml]: build numbers always starts with zero.

File: recipes/taxator-tk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  url: https://research.bifo.helmholtz-hzi.de/webapps/wa-download/download.php?id=v4Xdsvn6ib6AYQAyXmcTz6
+  md5: 6575138bb60503e9fdb76ff463be9df3
+
+build:
+  number: 1
+  skip: True # [osx]
+
+requirements:
+  build: 
+    - {{ compiler('c') }}
```
Comment (rvalieris) [inline recipes/taxator-tk/meta.yaml]: nothing is being compiled so this is not necessary.
but do consider compiling it on the recipe if possible, its always better that way.

**Comment by dpryan79** [comment]
Looks like this got accomplished in another PR

**Pull request closed without merging**